### PR TITLE
update trininty to the latest version

### PR DESCRIPTION
**Pull request by bgruening**
There is some hope that this version clobbers a bit less the disk and reduces the memory requirements.

### CODE REVIEW

File: tools/trinity/trinity.xml [hunk 1]
```diff
@@ -1,12 +1,12 @@
...
     <description>de novo assembly of RNA-Seq data</description>
     <macros>
         <import>macros.xml</import>
     </macros>
     <expand macro="bio_tools"/>
     <expand macro="requirements">
-        <requirement type="package" version="8.32">coreutils</requirement>
-        <requirement type="package" version="3.2.3">rsync</requirement>
+        <requirement type="package" version="3.2.7">rsync</requirement>
+        <requirement type="package">perl-db_file</requirement>
```
Comment (bgruening) [inline tools/trinity/trinity.xml:9]:
```suggestion
```

Comment (bgruening) [reply tools/trinity/trinity.xml:9]: Fixing this in conda

File: tools/trinity/align_and_estimate_abundance.xml [hunk 2]
```diff
@@ -152,35 +151,31 @@
...
-        <data format="tabular" name="genes_counts_salmon" label="${tool.name} on ${on_string}: genes counts" from_work_dir="output/quant.sf.genes">
+        <data format="tabular" name="genes_counts_salmon" label="${tool.name} on ${on_string}: genes counts (Salmon)" from_work_dir="output/quant.sf.genes">
             <filter>method['est_method'] == "salmon"</filter>
         </data>
 
-        <data format="tabular" name="isoforms_counts_kallisto" label="${tool.name} on ${on_string}: isoforms counts" from_work_dir="output/abundance.tsv">
+        <data format="tabular" name="isoforms_counts_kallisto" label="${tool.name} on ${on_string}: isoforms counts (Kallisto)" from_work_dir="output/abundance.tsv">
             <filter>method['est_method'] == "kallisto"</filter>
         </data>
-        <data format="tabular" name="genes_counts_kallisto" label="${tool.name} on ${on_string}: genes counts" from_work_dir="output/abundance.tsv.genes">
```
Comment (bernt-matthias) [inline tools/trinity/align_and_estimate_abundance.xml:183]: Maybe we can `expect_num_outputs` to the tests?

**Review by bernt-matthias — COMMENTED** [review]
https://github.com/trinityrnaseq/trinityrnaseq/releases/tag/v2.11.0 introduced `--no_parallel_norm_stats` which seem interesting .. up to you :)

**Review by bernt-matthias — APPROVED** [review]

**Pull request merged**